### PR TITLE
Mir-prefer a new recipe

### PR DESCRIPTION
**Pull request by mictadlo**
* [x] I have read the guidelines above.
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by bgruening** [comment]
There is no new recipe included here.

**Comment by mictadlo** [comment]
@bgruening merging?

File: recipes/mir-prefer/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+  run:
+    - python 
+    - viennarna
+    - samtools ==0.1.19
+    - bowtie ==1.2.0
+
+test:
+  commands:
+    - bowtie-align-reads.py -h 2>&1 | grep "python bowtie-align-reads.py" > /dev/null
```
Comment (bgruening) [inline recipes/mir-prefer/meta.yaml]: In general you don't need this /dev/null and it will see in the error logs more.

**Comment by bgruening** [comment]
LGTM! Thanks a bunch! Can you please also push changes upstream here!

**Comment by mictadlo** [comment]
How to I get them upstream?

**Pull request merged**